unique_sdk/unique_sdk/_http_client.py

```diff
@@ -16,21 +28,59 @@ def new_default_http_client(*args, **kwargs) -> "HTTPClient":
     return impl(*args, **kwargs)
 
 
+def new_http_client_async_fallback(*args, **kwargs) -> "HTTPClient":
+    if aiohttp:
+        impl = AIOHTTPClient
+    elif httpx:
+        impl = HTTPXClient
```

Comment:
Ok I now see that this method is used to find a fallback client as well. This method here needs an else statement to raise a meaning error.